src/application/polymarket/hooks/use-warpcast-markets.ts

```diff
@@ -1,8 +1,7 @@
 import { useQuery } from '@tanstack/react-query';
 import { useMemo } from 'react';
 
-import { useWarpcastExternalLinksPooling } from 'src/host/warpcast';
-
+import { useWarpcastExternalLinksPooling } from '../../../host/warpcast';
```

Comment:
for imports across different layers like application->host we should use absolute imports, `host/warpcast` should work